pom.xml

```diff
@@ -11,8 +11,8 @@
     <packaging>pom</packaging>
 
     <properties>
-        <source.version>1.8</source.version>
-        <target.version>1.8</target.version>
+        <source.version>11</source.version>
```

Comment:
Are you doing something that requires 11?

Reply:
well requires JDK9 -- I am using the StackWalker api, plus since we are targeting 11 we should be executing against IMO